dashboard/src/app/workspaces/list-workspaces/list-workspaces.controller.ts

```diff
@@ -18,21 +19,25 @@
  */
 export class ListWorkspacesCtrl {
 
+  cheNamespaceRegistry: CheNamespaceRegistry;
```

Comment:
I've noticed that almost all new variables that you've added are using private keyword but not this one.